src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Providers/Providers/AzureSqlPsBackupProvider.cs

```diff
@@ -188,7 +188,25 @@ public ProtectionPolicyResponse ModifyPolicy()
 
         public List<Models.AzureRmRecoveryServicesBackupContainerBase> ListProtectionContainers()
         {
-            throw new NotImplementedException();
+            string name = (string)this.ProviderData.ProviderParameters[ContainerParams.Name];
+
+            ProtectionContainerListQueryParams queryParams = new ProtectionContainerListQueryParams();
+
+            queryParams.BackupManagementType = Microsoft.Azure.Management.RecoveryServices.Backup.Models.BackupManagementType.AzureSql.ToString();
```

Comment:
Yes, that's what my comment was about :) Please use alias instead of the complete namespace